glightning/plugin.go

```diff
@@ -151,7 +151,7 @@ type OpenChannelEvent struct {
 	HtlcMinimumMillisatoshis          string `json:"htlc_minimum_msat"`
 	//FeeratePerKw ..//??
 	ToSelfDelay      uint16 `json:"to_self_delay"`
-	MaxAcceptedHtlcs uint16 `json"max_accepted_htlcs"`
```

Comment:
while a valid fix, this change is not related to the rest of the commit. can you pull it out into a separate PR/commit?

Reply:
I removed and pushed, build fail, I will make a separate PR